docs/validationtest/README.md

```diff
@@ -1,8 +1,19 @@
-# Subscriber
+# Validationtest
 
-This tool is used to create Kyma subscriptions and their corresponding subscribers.
+This set of tools is used to publish continues legacy and cloudevents the Eventing publisher proxy Kyma. In addition to the publisher
+a set of subscribers is created with corresponding subscriptions.
 
-## Eventing infrastructure
+## Application to event type mapping
+
+Currently, it is hardcoded which application sends which event type as follows:
+
+| Application   | Event type                                       |
+|---------------|--------------------------------------------------|
+| commerce      | order.created                                    |
+| appname       | DocuSing_BO.Account_DocuSign.Updated             |
+| no-app        | New.Some-Other.Order-äöüÄÖÜβ.Final.C-r-e-a-t-e-d |
+
+## Subscription
 
 1. **Three** microservice subscribers.
 2. **Three** Kyma function subscribers.
```

Comment:
Can't comment on line 20, but I suggest the following punctuation:

`3. **Six** Kyma subscriptions; each of them has the following hardcoded filters:`